.github/workflows/test.yaml

```diff
@@ -9,18 +9,18 @@ on:
       - development
       - master
       - release
-    
+      - rnn
```

Comment:
`rnn` branch will be deleted -> delete
`release` branch not used anymore in workflow -> delete